cars/src/main/java/ro/sci/cars/Car.java

```diff
@@ -1,104 +0,0 @@
-package ro.sci.cars;
-
-import ro.sci.cars.util.FuelType;
-
-/**
- * Created by alex on 5/11/17.
- */
-public class Car implements Vehicle {
-
-    private int fuelTankSize;
-    private int gears;
-    private double availableFuel;
-    private FuelType fuelType;
-    private double factoryAvgConsumptionPer100km;
-    private double driveAvgConsumptionPer100km;
-    private int currentGear;
-
-    public Car(double fuel, FuelType fuelType, int fuelTankSize, int gears, double avgConsumption) {
-        this.availableFuel = fuel;
-        this.fuelType = fuelType;
-        this.fuelTankSize = fuelTankSize;
-        this.gears = gears;
-        this.factoryAvgConsumptionPer100km = avgConsumption;
-    }
-
-    public void start() {
-        System.out.println("start");
-    }
-
-    public void stop() {
-        System.out.println("stop");
-    }
-
-    public void drive(double km) {
```

Comment:
this should not be here. this is a Model class. All methods like this one should be in service classes